machine/qemu/init.go

```diff
@@ -267,7 +267,7 @@ func init() {
 	// gob.Register(QemuDevicePcTestdev{})
 	// gob.Register(QemuDevicePciTestdev{})
 	// gob.Register(QemuDevicePcm3680Pci{})
-	// gob.Register(QemuDevicePvpanic{})
+	gob.Register(QemuDevicePvpanic{})
```

Comment:
Lingering from another PR?

Reply:
how did this happen. I'm never reordering commits again, so strange